### ISSUE TITLE
Add URL links for every article 

### ISSUE THREAD

**Issue by KiranPranay** [ServiceToMankind/OpenCodeLaw#7]
Add a button to easily copy the URL of the article.

**Comment by Surya-V-28** [comment]
Please assign me this i would like to work on it ,

**Comment by Surya-V-28** [comment]
@deepak4566  , do you want me to create the link and when its copied we can show a small toast message ? 


or  can you describe a little more about it  ?

**Comment by KiranPranay** [comment]
> @deepak4566 , do you want me to create the link and when its copied we can show a small toast message ?
> 
> or can you describe a little more about it ?

Small toast message is enough. Since "copy URL" buttons are already implemented.

**Comment by Surya-V-28** [comment]
yeah ,  I will work on it

**Comment by Surya-V-28** [comment]
if there any changes need let me know

**Comment by KiranPranay** [comment]
> @KiranPranay @deepak4566
> 
> do we allowed to use the icons ? here is the sample i worked on , let me know if there is any issues
> 
> ![image](https://user-images.githubusercontent.com/102609082/273387843-6aeed250-851b-4fa2-9735-ad979b52defe.png)

Yes, Icons are welcome. Try to use svg icons and border on focus.

**Comment by Surya-V-28** [comment]
@KiranPranay , i will change to svg icons

**Comment by KiranPranay** [comment]
> change

sure 👍🏼
